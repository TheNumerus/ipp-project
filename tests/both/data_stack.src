.IPPcode20
DEFVAR GF@stack_top
PUSHS bool@false
POPS GF@stack_top
DEFVAR GF@stack_type
TYPE GF@stack_type GF@stack_top
WRITE string@na\032stacku\032je\032
WRITE GF@stack_type
WRITE string@\032s\032hodnotou:\032
WRITE GF@stack_top
WRITE string@\010
DEFVAR GF@str
PUSHS int@4096
MOVE GF@str string@Tohle\032je\032string\032a\032cislo:\032
PUSHS GF@str
WRITE string@Cislo\032a\032string:\010
POPS GF@stack_top
WRITE GF@stack_top
POPS GF@stack_top
WRITE GF@stack_top
WRITE string@\010