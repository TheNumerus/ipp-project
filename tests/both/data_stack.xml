<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode20"><instruction order="1" opcode="DEFVAR"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="2" opcode="PUSHS"><arg1 type="bool">false</arg1></instruction><instruction order="3" opcode="POPS"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="4" opcode="WRITE"><arg1 type="string">na\032stacku\032je:\032</arg1></instruction><instruction order="5" opcode="WRITE"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="6" opcode="WRITE"><arg1 type="string">\010</arg1></instruction><instruction order="7" opcode="DEFVAR"><arg1 type="var">GF@str</arg1></instruction><instruction order="8" opcode="PUSHS"><arg1 type="int">4096</arg1></instruction><instruction order="9" opcode="MOVE"><arg1 type="var">GF@str</arg1><arg2 type="string">Tohle\032je\032string\032a\032cislo:\032</arg2></instruction><instruction order="10" opcode="PUSHS"><arg1 type="var">GF@str</arg1></instruction><instruction order="11" opcode="WRITE"><arg1 type="string">Cislo\032a\032string:\010</arg1></instruction><instruction order="12" opcode="POPS"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="13" opcode="WRITE"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="14" opcode="POPS"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="15" opcode="WRITE"><arg1 type="var">GF@stack_top</arg1></instruction><instruction order="16" opcode="WRITE"><arg1 type="string">\010</arg1></instruction></program>
