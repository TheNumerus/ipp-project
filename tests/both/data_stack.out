na stacku je: false
Cislo a string:
Tohle je string a cislo: 4096
